src/main/resources/messages/en.yml

```diff
@@ -269,20 +269,36 @@ MinecraftPlayerDeathMessage:
       Text: ""
       IconUrl: ""
     Timestamp: false
-
-# Minecraft -> Discord achievement/advancement messages
 #
 # Available placeholders:
 # %achievement%: title of the achievement/advancement
-# %displayname%: display name from things like nicknames
-# %username%:    raw player username
 # %world%:       the name of the world the user is in
-# %worldalias%:  alias of world player is in via Multiverse-Core
-# %date%:        current date & time
-# PlaceholderAPI placeholders are also supported
 #
-MinecraftPlayerAchievementMessagesFormat: ":medal: **%displayname% has made the advancement %achievement%!**"
-
+MinecraftPlayerAchievementMessage:
+  Enabled: true
+  Webhook:
+    Enable: false
+    AvatarUrl: "%botavatarurl%"
+    Name: "%botname%"
+  Content: ""
+  Embed:
+    Enabled: true
+    Color: "#ffd700"
+    Author:
+      ImageUrl: "%embedavatarurl%"
+      Name: "%username% has made the advancement %achievement%!"
+      Url: ""
+    ThumbnailUrl: ""
+    Title:
+      Text: ""
+      Url: ""
+    Description: ""
+    Fields: []
+    ImageUrl: ""
+    Footer:
+      Text: ""
+      IconUrl: ""
+    Timestamp: false
```

Comment:
There should be a empty line after this line (in all languages)

Reply:
Added to all messages.yml languages